spk/sonarr/src/wizard/install_uifile.sh

```diff
@@ -0,0 +1,115 @@
+#!/bin/bash
+
+function incompatible_upgrade
+{
+    cat <<EOF >"${SYNOPKG_TEMP_LOGFILE}"
+[{
+	"step_title": "ATTENTION! Incompatible upgrade detected",
+	"invalid_next_disabled_v2": true,
+	"items": [{
+		"type": "textfield",
+		"desc": "<strong style='color:red'>WARNING:</strong> This update is an upgrade to Sonarr v4 from Sonarr v3. We have detected that you have Sonarr v2 installed. We suggest that you upgrade your previous installation to Sonarr v3 before installing this upgrade.<br><br><b>Installation will abort.</b>",
```

Comment:
Couldn't you simply tell something like, indeed incompatible as such if you continue your local legacy configs won't be ported over to the new installation and old install be deleted?  Just a thought...

Reply:
I would prefer that users take the decision to uninstall the old version on their own rather than doing it for them and losing their library data.